tools/deb/istio-iptables.sh

```diff
@@ -101,46 +109,47 @@ if [ -z "${ENVOY_UID:-}" ]; then
   ENVOY_UID=${ENVOY_UID},0
 fi
 
-# Remove the old chains, to generate new configs.
```

Comment:
I think this was important - in the VM case if a new env file is pushed ( or you edit), and 'systemctl restart ...' is called, 
without this clean it would fail (or result in overlaps or old setting still present ).